src/components/TableForm/TableForms.js

```diff
@@ -101,105 +116,106 @@ export default function TableForms() {
             ...confirmDialog,
             isOpen: false
         })
-        calendarTask.deleteTask(id); 
+        calendarTask.deleteTask(id);
         setRecords(calendarTask.getAllTasks())
         setNotify({
             isOpen: true,
             message: 'Deleted Successfully',
             type: 'error'
-        })        
+        })
     }
 
     return (
-        <>
-        <Paper className={classes.pageContent}>
-            <Toolbar>
-        <Controls.Input
-            label="Search Task"
+        <div className= 'container'>
+        <Paper className={classes.pageContent} >
+                <Toolbar>
+                    <Controls.Input
+                        label="Search Task"
             className = {classes.searchInput}
```

Comment:
Would this line need more spaces?

Reply:
I can't understand your question. There is no issue :)